client/components/post-schedule/clock.jsx

```diff
@@ -38,17 +38,23 @@ function checkTimeValue( value ) {
 }
 
 class PostScheduleClock extends Component {
+	constructor() {
+		super( ...arguments );
+
+		// bounds
+		this.adjustHour = this.handleKeyDown.bind( this, 'hour' );
```

Comment:
This might be a good place to use some es6 instead of `bind`
```js
this.adjustHour = event => this.handleKeyDown( 'hour', event );
this.adjustMinute = event => this.handleKeyDown( 'minute', event );
```